docs/messaging-implementation.md

```diff
@@ -54,6 +55,7 @@ point uw-frame to your desired feed.
 - **expireDate**: *(optional)* Accepts a simple ISO date, including time (as pictured). This is used to stop displaying a message at a certain day/time.
 - **featureImageUrl**: *(optional)* Used by popup announcements and announcements on the Features page.
 - **priority**: Set to "high" if you want the message to be displayed with higher visibility (i.e. As a priority notification or popup announcement, respectively).
+- **isSticky**: *(experimental)* Set to true if you want to disallow users from dismissing a notification. This should only be used for truly critical messages. 
```

Comment:
Consider documenting default behavior.  Maybe "By default users can dismiss notifications. Set to true to disallow user dismissing for this notification."

"This should only be used for truly critical messages. ": I'm not sure we know that this is the guidance yet. Stickiness may be less about criticality and more about does the notification dismiss itself when the underlying business process is fulfilled. If we can tell that you no longer have a library fine preventing your graduating, we should yank the notification for you when you pay the fine. Maybe how well we can automate this for the user drives how much dismissing the notification work we ought to expose to the user.

(Notifications are like a to do list where the items check themselves off when you do them?)

We might get to guidance that whether user dismissing should be disabled depends only on the criticality of the message. I think the point of the prospective user research is to figure this out.